### PR TITLE
Speed up `scripts/format-stylish.sh`

### PR DESCRIPTION
**Pull request by wenkokke**
# Description

- Set `exit_code` to `error_on_format`.
- Batch formatting with 50 files at a time.

### CODE REVIEW

File: scripts/format-stylish.sh
Comment (jorisdral) [inline scripts/format-stylish.sh:21]: Does batching make a difference?

File: scripts/pre-commit.sh
Comment (jorisdral) [inline scripts/pre-commit.sh:22]: Why is this commented out now?

**Review by jorisdral — APPROVED** [review]

File: scripts/pre-commit.sh
Comment (mheinzel) [inline scripts/pre-commit.sh:22]: What's the reason for commenting this out?

Comment (mheinzel) [reply scripts/pre-commit.sh:22]: Ah, already asked :smile:

Comment (wenkokke) [reply scripts/pre-commit.sh:22]: Oops!

File: scripts/format-stylish.sh
Comment (wenkokke) [reply scripts/format-stylish.sh:21]: Yes. Massive. In fact, it's the only thing that made a difference.

File: scripts/pre-commit.sh
Comment (wenkokke) [reply scripts/pre-commit.sh:22]: Oops!

**Comment by jorisdral** [comment]
Could you squash the commits before I press the merge button?

**Comment by wenkokke** [comment]
Closed in favour of #465.

**Pull request closed without merging**
